notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1251 @@
+{
```

Comment:
<p>Proposed wording:</p><p> </p><p>In this tutorial, you will create two classification models using Vertex AI TabNet Tabular Workflows. Each workflow is a managed instance of <a href="https://cloud.google.com/vertex-ai/docs/pipelines/introduction" target="_blank">Vertex Pipelines</a>. The first pipeline, TabNet CustomJob, is the simplest, with the fewest parameters to configure. The second pipeline, TabNet HyperparameterTuningJob, allows you to get the best set of hyperparameters for your dataset.</p><p> </p><p>After training, each pipeline return a link to the Vertex Model UI. You can use the UI to deploy the model, get online predictions, or run batch prediction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thanks, done.